templates/grocery_planner/gp1-meal-choice.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block css %}
+<link rel="stylesheet" href="{% static 'css/grocery-planner/gp1-meal-choice.css' %}">
+{% endblock %}
+
+{% block content %}
+<form>
+    <div class="jumbotron">
+        <div class="card text-white bg-success mb-3 mx-auto" id='page-title'>
+            <h1 class="card-header">Meal Choice</h1>
+        </div>
+        <table class="table table-hover">
+            <thead>
+                <tr>
+                    <th scope="col" class='meal-name-column'>Meal</th>
```

Comment:
Curious, What else is possible as scope?

Reply:
<img width="635" alt="Screen Shot 2020-10-05 at 9 33 05 AM" src="https://user-images.githubusercontent.com/42154517/95123519-cc53ec80-06ed-11eb-9a3e-33239682f162.png">